RCSwitch.cpp

```diff
@@ -457,6 +477,64 @@ char* RCSwitch::getCodeWordD(char sGroup, int nDevice, boolean bStatus){
 }
 
 /**
+ * Get coding for Type E (RSL devices sold by Conrad)
+ * source: http://www.mikrocontroller.net/topic/252895
+ *
+ * @param sGroup   Number of the switch group (1..4)
+ * @param nDevice  Number of the switch itself (1..4)
+ * @param bStatus  Whether to switch on (true) or off (false)
+ */
+char* RCSwitch::getCodeWordE(int sGroup, int nDevice, boolean bStatus) {
+    int nReturnPos = 0;
+    static char sReturn[32];
```

Comment:
hmmm... shouldn't this be 33? You write 2+2+4+8 = 32 values into this array -- and then add a 0 byte as string terminator.